marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/flow/reorderFlow.sjs

```diff
@@ -0,0 +1,67 @@
+const flowService = require("../lib/flowService.sjs");
```

Comment:
This test can might as well be called updateFlow.sjs as you have tests for updating description as well as the step order.